dev/io.openliberty.mail.2.0.internal_fat/build.gradle

```
@@ -14,6 +14,7 @@ dependencies {
   requiredLibs project(':com.ibm.ws.org.slf4j.api'),
                project(':com.ibm.ws.org.slf4j.jdk14'),
                'com.ibm.ws.com.icegreen:greenmail:2.0.0.fd826131f671b0d3230b0d0e1638d8c4',
+               'org.eclipse.angus:angus-mail:1.0.0',
```

```suggestion
               project(':io.openliberty.org.eclipse.angus.mail.2.1'),
```
Better to use our existing bundle ... but now I look at that bundle, the name looks wrong. It is version 1.0.0 rather than 2.1 (2.1 is the spec version). If you get a chance then I would rename `io.openliberty.org.eclipse.angus.mail.2.1` to just `io.openliberty.org.eclipse.angus.mail` and double check the package versions that it exports?